tests/support/ArrayzyTestTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+trait ArrayzyTestTrait
+{
+  public function simpleArrayProvider()
```

Comment:
Why are you what create a trait for a single method? This changes doesn't relate to the find method implementation.